hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/downloader/block/BlockFileTransformerTest.java

```diff
@@ -173,6 +176,148 @@ void unknownTransform() {
+                .returns(null, RecordItem::getPrevious)
+                .extracting(RecordItem::getTransactionRecord)
+                .returns(expectedTransactionHash, TransactionRecord::getTransactionHash));
+    }
+
+    @Test
+    void fileDeleteTransform() {
+        // given
+        var expectedRecordItem = recordItemBuilder
+                .fileDelete()
+                .recordItem(r -> r.hapiVersion(HAPI_VERSION))
+                .build();
+        var expectedTransactionHash = getExpectedTransactionHash(expectedRecordItem);
+        var blockItem = blockItemBuilder.fileDelete(expectedRecordItem).build();
+        var blockFie = blockFile(List.of(blockItem));
+
+        // when
+        var recordFile = blockFileTransformer.transform(blockFie);
+
+        // then
+        assertRecordFile(recordFile, blockFie, items -> assertThat(items)
+                .hasSize(1)
+                .first()
+                .satisfies(item -> assertRecordItem(item, expectedRecordItem))
+                .returns(null, RecordItem::getPrevious)
+                .extracting(RecordItem::getTransactionRecord)
+                .returns(expectedTransactionHash, TransactionRecord::getTransactionHash));
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = RecordItemBuilder.SuccessfulStatusCreateFile.class)
+    void fileCreateTransform(RecordItemBuilder.SuccessfulStatusCreateFile successfulStatus) {
+        // given
+        var expectedRecordItem = recordItemBuilder
+                .fileCreate()
+                .recordItem(r -> r.hapiVersion(HAPI_VERSION))
+                .receipt(r -> r.setStatus(successfulStatus.toResponseCodeEnum()))
+                .build();
+        var expectedTransactionHash = getExpectedTransactionHash(expectedRecordItem);
+        var blockItem = blockItemBuilder.fileCreate(expectedRecordItem).build();
+        var expectedFileId = blockItem
+                .stateChanges()
+                .getFirst()
+                .getStateChanges(3)
+                .getMapUpdate()
+                .getKey()
+                .getFileIdKey()
+                .getFileNum();
+        var blockFie = blockFile(List.of(blockItem));
+
+        // when
+        var recordFile = blockFileTransformer.transform(blockFie);
+
+        // then
+        assertRecordFile(recordFile, blockFie, items -> assertThat(items)
+                .hasSize(1)
+                .first()
+                .satisfies(item -> assertRecordItem(item, expectedRecordItem))
+                .returns(null, RecordItem::getPrevious)
+                .extracting(RecordItem::getTransactionRecord)
+                .returns(expectedTransactionHash, TransactionRecord::getTransactionHash)
+                .returns(
+                        expectedFileId,
+                        transactionRecord ->
+                                transactionRecord.getReceipt().getFileID().getFileNum()));
+    }
+
+    @Test
+    void fileCreateTransform_whenStatusIsNotSuccess() {
```

Comment:
Same comment in all tests. Don't use underscores in method names.

```suggestion
    void fileCreateTransformWhenStatusIsNotSuccess() {
```

Reply:
Fixed

Reply:
Still has underscore

Reply:
Sorry, totally missed this one lone test.